routes/users.js

```diff
@@ -41,18 +43,23 @@ router.post("/register", function(req,res){
 
 			if(user){
 				if(user.username === username){
-					req.flash('error_msg', 'Username is already in use');
+					req.flash('error_msg', 'Username is already in use ');
 				}
 				if(user.email === email) {
 					req.flash('error_msg', 'Email address is already in use');
 				}
 				res.redirect('/users/register');
 			} else{
+				var seed = crypto.randomBytes(32).toString('hex');
```

Comment:
some documentation on the choice is required. 
Why 32 magic number or 9 the line below. Be more descriptive when it come to this details and don't hard-code numbers.